x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.js

```diff
@@ -49,6 +49,37 @@ export class HeadlessChromiumDriverFactory {
         });
 
         page = await chromium.newPage();
+        await page.setRequestInterception(true);
+
+        page.on('request', req => {
+          this.logger.debug(`Request: [${req.url()}]`);
+          req.continue();
+        });
+        page.on('response', res => {
+          this.logger.debug(`Response: [${res.url()} - ${res.status()}]`); // TODO: test more this logs more than just 200 responses
+        });
+        page.on('requestfailed', async failedReq => {
```

Comment:
This is producible by leaving out the `req.continue()` in the `request` interceptor above. In that scenario, the fail message will be `net::ERR_ABORTED`